package.json

```diff
@@ -14,7 +14,11 @@
   },
   "dependencies": {
     "dotenv": "^16.3.1",
+    "i": "^0.3.7",
+    "npm": "^10.2.3",
```

Comment:
what is `npm`?

I'm assuming you accidentally typed in the terminal `npm i npm i ...`